pkg/broker/broker.go

```diff
@@ -439,11 +450,21 @@ func (a AnsibleBroker) Bind(instanceUUID uuid.UUID, bindingUUID uuid.UUID, req *
 	}
 
 	bindExtCreds := &apb.ExtractedCredentials{Credentials: make(map[string]interface{})}
+	var podName string
 	if a.brokerConfig.LaunchApbOnBind {
-		bindExtCreds, err = apb.Bind(instance, &params, a.clusterConfig, a.log)
+		a.log.Info("Broker configured to run APB bind")
+		a.log.Info("Starting APB bind...")
+		podName, bindExtCreds, err = apb.Bind(instance, &params, a.clusterConfig, a.log)
+
+		sm := apb.NewServiceAccountManager(a.log)
+		a.log.Info("Destroying APB sandbox...")
+		sm.DestroyApbSandbox(podName, instance.Context.Namespace)
+
 		if err != nil {
 			return nil, err
 		}
+	} else {
+		a.log.Warning("Broker configured to *NOT* launch and run APB bind")
```

Comment:
I wanted to call attention to it so that people are not surprised if they expect their APB binds to run and the broker is configured to *not* run them.